en/04_Changelogs/5.0.0.md

```diff
@@ -1077,28 +1077,28 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
+- Removed deprecated method `SilverStripe\Security\Security::setDefaultAdmin()` - use [`SilverStripe\Security\DefaultAdminService::setDefaultAdmin($username, $password)`](api:SilverStripe\Security\DefaultAdminService::setDefaultAdmin()) instead
 - Removed deprecated method `SilverStripe\View\Requirements_Backend::getMinifier()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\View\Requirements_Backend::getMinifyCombinedFiles()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\View\Requirements_Backend::setMinifier()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\View\Requirements_Backend::setMinifyCombinedFiles()` without equivalent functionality to replace it
 - Removed deprecated method `SilverStripe\View\Shortcodes\EmbedShortcodeProvider::embedForTemplate()` - use `embeddableToHtml()` instead
-- Removed deprecated method `SilverStripe\View\SSViewer::set_theme()` - use `SSViewer::set_themes()` instead
+- Removed deprecated method `SilverStripe\View\SSViewer::set_theme()` - use `set_themes()` instead
 - Removed deprecated method `SilverStripe\View\SSViewer_BasicIteratorSupport::First()` - use `IsFirst()` instead
 - Removed deprecated method `SilverStripe\View\SSViewer_BasicIteratorSupport::Last()` - use `IsLast()` instead
-- Removed deprecated method `SilverStripe\View\ViewableData::ThemeDir()` - use `ModuleResourceLoader::resourcePath()` or `ModuleResourceLoader::resourceURL()` instead
+- Removed deprecated method `SilverStripe\View\ViewableData::ThemeDir()` - use [`SilverStripe\Core\Manifest\ModuleResourceLoader::resourcePath()`](api:SilverStripe\Core\Manifest\ModuleResourceLoader::resourcePath()) or [`SilverStripe\Core\Manifest\ModuleResourceLoader::resourceURL()`](api:SilverStripe\Core\Manifest\ModuleResourceLoader::resourceURL()) instead
 - Removed deprecated config `SilverStripe\Control\Director.alternate_public_dir` without equivalent functionality to replace it
 - Removed deprecated config `SilverStripe\Control\HTTP.cache_ajax_requests`
-- Removed deprecated config `SilverStripe\Control\HTTP.cache_control` - use `HTTPCacheControlMiddleware` instead
-- Removed deprecated config `SilverStripe\Control\HTTP.disable_http_cache` - use `HTTPCacheControlMiddleware.defaultState.defaultForcingLevel` instead
-- Removed deprecated config `SilverStripe\Control\HTTP.vary` - use `HTTPCacheControlMiddleware` instead
+- Removed deprecated config `SilverStripe\Control\HTTP.cache_control` - use [`SilverStripe\Control\Middleware\HTTPCacheControlMiddleware`](api:SilverStripe\Control\Middleware\HTTPCacheControlMiddleware) instead
+- Removed deprecated config `SilverStripe\Control\HTTP.disable_http_cache` - use `SilverStripe\Control\Middleware\HTTPCacheControlMiddleware.defaultState.defaultForcingLevel` instead
```

Comment:
`defaultState` and `defaultForcingLevel` are two separate configuration properties. The source code has a `/` between them:
https://github.com/silverstripe/silverstripe-framework/blob/7c5cbae928a336eb498bdce0e85ac3be3f997bf8/src/Control/HTTP.php#L49-L54

I think this is mean to mean "use `HTTPCacheControlMiddleware.defaultState` or `HTTPCacheControlMiddleware.defaultForcingLevel` instead"

Reply:
Done